### DISCUSSION TITLE
Eslint config & dprint

### DISCUSSION THREAD

**Discussion by linonetwo** [tiddly-gittly/Modern.TiddlyDev#46] [General]
df4a8c4 我加上了太记里用的高级 eslint 配置 eslint-config-tidgi ，并用了 dprint ，如 https://github.com/Jermolene/TiddlyWiki5/pull/7474

这个新的 eslint 配置检查出了不少之前没查出的问题，我顺便修了一下 6ac2622 。希望你不介意这个更高级复杂的 eslint 配置 @Gk0Wk

**Comment by Gk0Wk** [comment]
行hhhh但看起来主要是一些代码风格的改变。

另外Node和Element有啥区别么

**Reply by linonetwo** [reply]
不知道呀，反正 eslint 说他们不一样，就顺便改了，我也懒得想，有机器想了，人就不用思考了…脑力劳动机械化与科学技术现代化嘛

**Reply by linonetwo** [reply]
https://github.com/tiddly-gittly/eslint-config-tidgi 里用的插件还是挺多的
